src/components/ProgressCircle.ts

```diff
@@ -116,6 +121,15 @@ export class ProgressCircle extends Component<ProgressCircleProps, { alertMessag
                 progressText = `${value}`;
             } else if (text === "percentage") {
                 progressText = progress + "%";
+            } else if (text === "static" && this.props.displayTextStatic) {
+                progressText = !(this.props.displayTextStatic.includes("{value}"))
```

Comment:
What if it doesn't include `{value}` but includes `{maximumValue}`?

This check is unnecessary... just do the usual string replace for the template strings. If no templates exist, it won't hurt the string.

Reply:
Removing the check will display progress value twice if the datasource is attribute because we are prepending `progress` at every check so with that check of include `{value}` or `{maximumValue}` we are able to control how the display text will look like.

Reply:
`String.include` is not save to use in IE 11 see https://caniuse.com/#search=includes

Reply:
Removed